mobile/test/common/integration/base_client_integration_test.cc

```diff
@@ -249,6 +249,38 @@ testing::AssertionResult BaseClientIntegrationTest::waitForCounterGe(const std::
   return testing::AssertionSuccess();
 }
 
+uint64_t BaseClientIntegrationTest::getGaugeValue(const std::string& name) {
```

Comment:
we don't need these changes, BaseClientIntegrationTest already has these functions

Reply:
BaseClientIntegrationTest has similar functions getCounterValue and waitForCounterGe, but not the gauge equivalents.

Reply:
ah you're right, missed that, thanks for adding it!